python/taulabs/uavo.py

```diff
@@ -80,6 +85,29 @@ def from_bytes(cls, data, timestamp, instance_id, offset=0):
 
         return cls._make(field_values)
 
+    def __repr__(self):
+        """ String representation of the contents """
+        rep = self.__class__.__name__ + '('
+        for field in self._fields:
+            field_value = getattr(self, field)
+            # Show ENUMs in a fancy way (string and value)
+            if hasattr(self, 'ENUM_' + field):
```

Comment:
Just to make sure, you are suggesting to add a list / tuple with the same length as `_fields` that also contains the enums (and `None` for things that are not enums)? Performance-wise, I'm not sure if this is needed, as the `__repr__` method is only used when printing objects and AFAIK `hasattr` is quite fast.